coalib/bearlib/aspects/Spelling.py

```diff
@@ -1,72 +1,68 @@
+        Words should always be written as they are supposed to be,
+        standartization facilitates communication.
         """
         fix_suggestions = """
-        Use coala :) to detect what word is not written as expected, given
-        that there is a bear for it ;).
+        Use the correct spelling for the misspelled words.
         """
 
 
 @Spelling.subaspect
-class aspectsYEAH:
+class GeneralWordSpelling:
     """
-    This aspect dictates that the term ``aspects`` and ``aspect`` must have
-    all letters in lower case and the term ``aspectsYEAH`` must match the
-    exact expression.
+    ``GeneralWordSpelling`` refers valid language's words spelling.
     """
     class docs:
         example = """
-        Valid Cases:
-        # aspects are the mother of all futures of coala
-        # ``aspectsYEAH`` project is simply awesome
-        Invalid Cases:
-        # Aspects are the mother of all futures of coala
-        # ``aspectsyeah`` project is simply awesome
+        This is toatly wonrg.
         """
-        example_language = 'All'
+        example_language = 'English'
         importance_reason = """
-        The concepts of aspects in coala are too fundamental to contain any
-        upper-case letters.
+        Good spelling facilitates communication. By following the same rules
+        for spelling words, we can all understand the text we read;
+        Good spelling confusion. Poor spelling distracts the reader and they
+        lose focus.
         """
         fix_suggestions = """
-        ``aspects`` or ``aspect`` are always written with all lower-case
-        letters or the term ``aspectsYEAH`` must match the exact expression.
+        You can use a spell-checker to fix this for you or just ensure
+        yourself that things are well written.
         """
 
 
 @Spelling.subaspect
-class coalaCorrect:
+class CompanySpecificWordSpelling:
     """
-    This aspect dictates that the term ``coala`` must always be written with
-    with a lowercase `c`.
+    `CompanySpecificWordSpelling` refers companies' or organisations'
+    (like coala) specifid words' spelling.
     """
     class docs:
         example = """
-        Valid Cases:
-        # ``coala`` is so much fun!
-        # ``coalA`` hmm... not exactly ``coala`` but it is fine ;)
-        Invalid Cases:
-        # ``Coala`` hmm... this is wrong :(
-        # ``CoalA``... @cobot please do something about this :(
+        `Coala`, `Aspects`, `aspectYEAH`.
         """
-        example_language = 'All'
+        example_language = 'English'
         importance_reason = """
-        We like it this way ;).
+        There are words you want to be written as you want, like your
+        organisation's name.
         """
         fix_suggestions = """
-        Always write ``coala`` with a lower case ``c``.
+        Simply make sure those words match with what is provided by the
+        company.
         """
+    specific_word = Taste[str](
```

Comment:
yeah, we should probably implement a bear for that (that is a more generic verserion of the coalaBear taking  a list of reg_expr as parameter)